uc-crux-llvm/src/UCCrux/LLVM/Cursor.hs

```diff
@@ -185,12 +185,17 @@ ppCursor top =
     Field _fieldTypes idx cursor ->
       ppCursor top cursor <> PP.pretty ("." ++ show idx)
 
--- | A 'Selector' points to a spot inside an argument or global variable.
+-- | A 'Selector' points to a spot inside
+--
+-- * an argument,
+-- * a global variable, or
+-- * the manufactured return value from a \"skipped\" function
 --
 -- For documentation of the type parameters, see the comment on 'Cursor'.
 data Selector m (argTypes :: Ctx (FullType m)) inTy atTy
   = SelectArgument !(Ctx.Index argTypes inTy) (Cursor m inTy atTy)
   | SelectGlobal !L.Symbol (Cursor m inTy atTy)
+  | SelectReturn !L.Symbol (Cursor m inTy atTy)
```

Comment:
Is the symbol the name of the function that was skipped? If so, is it the case that we never try this with indirect calls, where we wouldn't be able to get that symbol?

Reply:
Great question - This works just fine for indirect calls because these `Selector`s are generated *inside* the relevant override, together with the returned value:

https://github.com/GaloisInc/crucible/pull/718/files#diff-e76f47d5b800a8b1d75a4bed25c18ba59ed5181cc7b4538079604b32c7f063f1R198